product-mini/platforms/nuttx/main.c

```diff
@@ -119,13 +121,15 @@ split_string(char *str, int *count)
 static void *
 app_instance_repl(wasm_module_inst_t module_inst)
 {
-    char *cmd = NULL;
-    size_t len = 0;
+    size_t len = 128;
+    char *cmd = malloc(len);
     ssize_t n;
 
-    while ((printf("webassembly> "), n = getline(&cmd, &len, stdin)) != -1) {
```

Comment:
Should check whether cmd is NULL before while?

Reply:
Thanks, I forget to force push null check to server
Done

Reply:
Thanks, I forget to force push null check to server

Done